libs/core/src/components/sage-chip/sage-chip.scss

```diff
@@ -0,0 +1,212 @@
+  --background-accent: var(--sage-color-purple-100);
+  --background-accent-dot: var(--sage-color-purple-300);
+  --background-accent-interactive: var(--sage-color-purple-200);
+  --background-success: var(--sage-color-sage-100);
+  --background-success-dot: var(--sage-color-sage-300);
+  --background-success-interactive: var(--sage-color-sage-200);
+  --background-warning: var(--sage-color-yellow-100);
+  --background-warning-dot: var(--sage-color-yellow-300);
+  --background-warning-interactive: var(--sage-color-yellow-200);
+  --box-shadow-focus: 0 0 0 4px var(--sage-color-primary-200);
+
+  --border-radius: var(--sage-border-radius-lg);
+  --font-size: var(--sage-font-size-body-sm);
+  --font-size-lg: var(--sage-font-size-body);
+  --font-weight: var(--sage-font-weight-medium);
+  --line-height: var(--sage-line-height-sm);
+  --padding-block-sm: calc(var(--sage-spacing-xxs) / 2);
+  --padding-block: var(--sage-spacing-xxs);
+  --padding-block-lg: 6px;
+  --padding-inline-sm: calc(var(--sage-spacing-xxs) / 2);
+  --padding-inline: 10px;
+  --padding-inline-lg: calc(var(--sage-spacing-md) / 2);
+  --padding-inline-xl: 14px;
+  --margin-block-xs: calc(var(--sage-spacing-xxs) / 2);
+  --margin-inline-sm: calc(var(--sage-spacing-xs) / 2);
+  --margin-inline-xs: calc(var(--sage-spacing-xxs) / 2);
+  --margin-inline: 6px;
+  --margin-inline-lg: var(--sage-spacing-xs);
+  --margin-inline-xl: calc(var(--sage-spacing-md) / 2);
+
+  --dot-size: 4px;
+  --close-size: 10px;
+  --close-size-lg: 24px;
+
+  align-items: center;
+  border-radius: var(--border-radius);
+  display: inline-flex;
+  font-size: var(--font-size);
+  font-weight: var(--font-weight);
+  line-height: var(--line-height);
+  padding-block: var(--padding-block-sm);
+  padding-inline: var(--padding-inline);
+}
+
+$sage-chip-sentiment: (
+  accent: var(--background-accent),
+  danger: var(--background-danger),
+  info: var(--background-info),
+  neutral: var(--background-neutral),
+  success: var(--background-success),
+  warning: var(--background-warning),
+);
+
+$sage-chip-sentiment-dots: (
+  accent: var(--background-accent-dot),
+  danger: var(--background-danger-dot),
+  info: var(--background-info-dot),
+  neutral: var(--background-neutral-dot),
+  success: var(--background-success-dot),
+  warning: var(--background-warning-dot),
+);
+
+$sage-chip-sentiment-interactive: (
+  accent: var(--background-accent-interactive),
+  danger: var(--background-danger-interactive),
+  info: var(--background-info-interactive),
+  neutral: var(--background-neutral-interactive),
+  success: var(--background-success-interactive),
+  warning: var(--background-warning-interactive),
+);
+
+@each $sentiment, $value in $sage-chip-sentiment {
+  :host(.sage-chip--#{$sentiment}) {
+    background: $value;
+
+    // dot colors
+    .sage-chip__dot {
+      /* stylelint-disable-next-line */
+      background: map-get($sage-chip-sentiment-dots, $sentiment);
```

Comment:
@ju-Skinner is there anything we can do here to prevent the linter from yelling about `map-get()`? Comments work for now, but should we consider a rule update?